src/Http/ApiClient.php

```diff
@@ -375,6 +378,21 @@ protected function determineBodyString($body): string
         return $bodyString;
     }
 
+    /**
+     * @param RequestException $exception
+     *
+     * @return bool
+     */
+    private function isCurlErrorCodeZero(RequestException $exception): bool
+    {
+        $allMatch = [];
+
+        preg_match(self::REGEX_CURL_ERROR_CODE, $exception->getMessage(), $allMatch);
+
+        return isset($allMatch[self::REGEX_NAMED_GOUP_ERROR_CODE]) &&
```

Comment:
Just a (personal preference) but I like the `&&` on the newline, this makes it easier to understand / spot the operators (in longer lists). But this is also fine, just my 2 cents ;)